package.json

```diff
@@ -25,6 +25,7 @@
     "husky": "^8.0.3",
     "lint-staged": "^13.3.0",
     "prettier": "^3.2.5",
-    "prettier-plugin-packagejson": "^2.4.14"
+    "prettier-plugin-packagejson": "^2.4.14",
+    "rollup": "^4.14.1"
```

Comment:
This is needed for the `package.config.ts` typings to pass, as vitest installs a different version of `rollup` than the one used by `@sanity/pkg-utils`